torch/csrc/jit/script/module.cpp

```diff
@@ -20,14 +35,20 @@ std::vector<Value*> Method::emit_call_to(SourceRange loc, Method & callee, Array
         << "' is called recursively involving this call site. Recursive calls are not supported";
   }
   auto fn = callee.graph();
-  ensureSizeMatches(loc, callee.num_inputs(), inputs.size(), "inputs");
-  std::vector<Value*> all_inputs = inputs;
+
+  std::stringstream failure_messages;
+  auto all_inputs = tryMatchSchema(
+    callee.schema ? *callee.schema : defaultSchemaFor(callee),
```

Comment:
When could it happen that a script function we're calling misses schema?

Reply:
Traces don't have schema, so defaultSchemaFor generates reasonable schema for them.